backend/src/main/resources/application.properties

```diff
@@ -37,19 +37,19 @@ spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
 own.bpnl=${OWN_BPNL:BPNL4444444444XX}
 
 # Own name (self-description)
-own.name=${OWN_NAME:Scenario Customer}
+own.name=${OWN_NAME:Control Unit Creator Inc.}
 
 # Own BPNS (optional: if this is set, then set own.site.name as well)
 own.bpns=${OWN_BPNS:BPNS4444444444XX}
 # Name of Site (see above)
-own.site.name=${OWN_SITE:Hauptwerk Musterhausen}
+own.site.name=${OWN_SITE:Control Unit Creator Production Site}
 
 # If a BPNS is set, then this BPNA will be attached to it.
 # Otherwise, it will be attached immediately to the BPNL (see above)
 own.bpna=${OWN_BPNA:BPNA4444444444AA}
-own.streetandnumber=${OWN_STREETANDNUMBER:Musterstrasse 35b}
-own.zipcodeandcity=${OWN_ZIPCODEANDCITY:77777 Musterhausen}
-own.country=${OWN_COUNTRY:Germany}
+own.streetandnumber=${OWN_STREETANDNUMBER:Wall Street 101}
```

Comment:
Would be great, if you applies these changes to the DataInjectionCommandLineRunner as well, in order to keep that Demo-Scenario Data coherent.

Reply:
updated